blog/comparing-sortings.md

```diff
@@ -0,0 +1,98 @@
+---
+slug: comparing-sorting-algorithms
+title: Comparing the different Sorting Algorithms
+authors: [ARYAN-JAIN]
+tags: [ARYAN-JAIN, algo, dsa, algorithms, sorting, time-complexity]
+---
+
+# Comparison of Sorting Algorithms
+
+Sorting algorithms are essential for organizing data, making it easier to analyze, search, and manipulate. Here’s an overview of some commonly used sorting algorithms, comparing their time complexity, efficiency, and applications.
+
+## 1. Bubble Sort
+**Description**: Bubble Sort repeatedly steps through the list, compares adjacent elements, and swaps them if needed until the list is sorted.
+
+- **Time Complexity**: \(O(n^2)\) average and worst-case; \(O(n)\) best-case (already sorted)
+- **Space Complexity**: \(O(1)\)
```

Comment:
replace `\(O(n^2)\)` with `$O(n^2)$`, `\(O(n)\)` with `$O(n)$` , and `\(O(1)\)` with `$\(O(1)\)$`